rfc/rfc017-distributed-network-grpc-v2.md

```diff
@@ -101,8 +102,9 @@ The protocol generally operates as follows:
    * When the XOR is the same: no action required.
    * When different:
       * Remove all known transaction references from the list.
-      * Request the remaining transactions.
-      * Bob calculates a new XOR value combining its own XOR value and the newly requested transactions hashes. If this new XOR value differs and if Alice's LC is equal or higher thatBob's highest LC value, Bob sends a `State` message as defined in Chapter 6. 
+      * Bob calculates a new XOR value combining its own XOR value and the remaining new/unknown transactions hashes. 
+      * If this new XOR value matches that of Alice, or if Alice's LC is lower, Bob requests the list of unknown transactions.
```

Comment:
This is independent of the LV value

Reply:
This is also part of the 1 conversation limit. The Gossip can only result in 1 new conversation, so a TxListQuery or a State message. 

If the new XORs match, all missing Tx should be requested to make the DAGs consistent.

If the new XORs still don't match, Bob should send a State message, but only if Alice is not behind. It doesn't make sense to also request missing Tx if that doesn't resolve their differences since these are likely to have missing prevs, and these missing Tx will be solved by in the State route as well. If Alice is behind, we could do nothing and require Alice to update everything first, or we can request the missing Txs from Alice we know Bob doesn't have. This describes the latter, which could also result in a State message if any of the requested Txs has missing prevs.

Reply:
The XOR can never match when the LC value is lower, it can be ignored?